aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -682,5 +696,75 @@ TORCH_IMPL_FUNC(_convert_indices_from_csr_to_coo_structured_cpu) (
   }
 }
 
+Tensor& linalg_solve_sparse_csr_out(const Tensor& input, const Tensor& other, Tensor& result) {
```

Comment:
@krshrimali you need to include `<ATen/ops/linalg_solve_native.h>` for the declaration of this function which includes the `TORCH_API` export macro.

Reply:
Thank you so much, @peterbell10 - this solves the failures! Appreciate the help :)